byron/ledger/formal-spec/blockchain-interface.tex

```diff
@@ -607,10 +608,9 @@ \subsection{Update-proposals interface}
       \right)}\\
       %
       {\begin{array}{r@{~\leteq~}l}
-        \var{stbl_{cps}} & \dom~(cps' \restrictrange [.., s_n - 2 \cdot k])\\
-        \var{stbl_{raus}} & \var{stbl_{cps}} \restrictdom \var{raus}\\
+        \var{cfm_{raus}} & \dom~(cps') \restrictdom \var{raus'}\\
```

Comment:
There's a `'` here and not in the executable spec, is that intentional?

Reply:
Yeah, in the case of the executable spec we do this:

```haskell
          ( (pv, pps)
            , fads
            , avs
            , rpus
            , raus
            , cps
            , vts
            , bvs
            , pws) = us'
```

and it felt weird to put a `'`' to each variable there. But maybe I should have ...

Reply:
I'm merging this for now, since I'm terrified of the hydra queue :scream_cat: